src/main/java/com/cburch/logisim/gui/chrono/PopupMenu.java

```diff
@@ -33,11 +33,12 @@ public PopupContents() {
       RadixOption radix = null;
       if (signals.size() > 0) {
         radix = signals.get(0).info.getRadix();
-        for (var i = 1; i < signals.size(); i++)
+        for (var i = 1; i < signals.size(); i++) {
```

Comment:
Maybe, consider using `++i` instead of `i++` to increment, as only the new value after incrementation is needed instead of the previous value.

Reply:
As this is a minor point, I mark it as resolved to allow merging of this PR.